# Occitan (post 1500) translation for linuxmint
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the linuxmint package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: linuxmint\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2016-11-02 13:58+0000\n"
"PO-Revision-Date: 2016-11-14 16:06+0000\n"
"Last-Translator: Cédric VALMARY (Tot en òc) <cvalmary@yahoo.fr>\n"
"Language-Team: Occitan (post 1500) <oc@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2017-05-06 12:34+0000\n"
"X-Generator: Launchpad (build 18366)\n"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:111
#: usr/lib/linuxmint/mintbackup/mintbackup.py:251
#: usr/lib/linuxmint/mintbackup/mintbackup.py:443
#: usr/lib/linuxmint/mintbackup/mintbackup.py:461
#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1499
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661 generate_desktop_files.py:26
#: generate_desktop_files.py:41
msgid "Backup Tool"
msgstr "Aisina de còpia de seguretat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:168
msgid "Preserve structure"
msgstr "Preservar la ierarquia dels fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:170
msgid ".tar file"
msgstr "Fichièr .tar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:171
msgid ".tar.bz2 file"
msgstr "Fichièr .tar .bz2"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:172
msgid ".tar.gz file"
msgstr "Fichièr .tar.gz"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:178
msgid "Never"
msgstr "Pas jamai"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:179
msgid "Size mismatch"
msgstr "Diferéncia dins las talhas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:180
msgid "Modification time mismatch"
msgstr "Diferéncia dins las datas de modificacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:181
msgid "Checksum mismatch"
msgstr "Diferéncia dins las signaturas sha1sum"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:182
msgid "Always"
msgstr "Totjorn"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:204
msgid "Excluded paths"
msgstr "Camins exclús"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:214
msgid "Type"
msgstr "Tipe"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:218
msgid "Detail"
msgstr "Detalh"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:224
#: usr/lib/linuxmint/mintbackup/mintbackup.py:233
msgid "Path"
msgstr "Camin"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:227
#: usr/lib/linuxmint/mintbackup/mintbackup.py:236
msgid "Error"
msgstr "Error"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:268
msgid "Store?"
msgstr "Emmagazinar ?"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:271
#: usr/lib/linuxmint/mintbackup/mintbackup.py:284
msgid "Name"
msgstr "Nom"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:281
msgid "Install"
msgstr "Installar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:289
msgid "Backup or Restore"
msgstr "Salvament o restabliment"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:290
msgid "Choose from the following options"
msgstr "Causissètz demest las opcions seguentas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:292
#: usr/lib/linuxmint/mintbackup/mintbackup.py:303
#: usr/lib/linuxmint/mintbackup/mintbackup.py:317
#: usr/lib/linuxmint/mintbackup/mintbackup.py:324
#: usr/lib/linuxmint/mintbackup/mintbackup.py:328
#: usr/lib/linuxmint/mintbackup/mintbackup.py:333
msgid "Backup files"
msgstr "Salvament de fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:293
#: usr/lib/linuxmint/mintbackup/mintbackup.py:337
#: usr/lib/linuxmint/mintbackup/mintbackup.py:347
#: usr/lib/linuxmint/mintbackup/mintbackup.py:353
#: usr/lib/linuxmint/mintbackup/mintbackup.py:358
msgid "Restore files"
msgstr "Restabliment de fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:294
#: usr/lib/linuxmint/mintbackup/mintbackup.py:362
#: usr/lib/linuxmint/mintbackup/mintbackup.py:367
#: usr/lib/linuxmint/mintbackup/mintbackup.py:373
#: usr/lib/linuxmint/mintbackup/mintbackup.py:378
msgid "Backup software selection"
msgstr "Salvament de la causida dels logicials"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:295
#: usr/lib/linuxmint/mintbackup/mintbackup.py:382
#: usr/lib/linuxmint/mintbackup/mintbackup.py:387
#: usr/lib/linuxmint/mintbackup/mintbackup.py:394
msgid "Restore software selection"
msgstr "Restabliment de la causida dels logicials"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:297
msgid "Make a backup of your files"
msgstr "Fasètz una còpia de salvament de vòstres fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:298
msgid "Save the list of installed applications"
msgstr "Fasètz una còpia de la lista dels logicials installats"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:299
msgid "Restore a previous backup"
msgstr "Restablissètz de fichièrs salvats"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:300
msgid "Restore previously installed applications"
msgstr "Restablissètz una lista de logicials salvada"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:304
msgid "Please select a source and a destination for your backup"
msgstr "Seleccionatz una font e una destinacion per vòstre salvament"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:305
#: usr/lib/linuxmint/mintbackup/mintbackup.py:341
#: usr/lib/linuxmint/mintbackup/mintbackup.py:349
msgid "Source:"
msgstr "Font :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:306
#: usr/lib/linuxmint/mintbackup/mintbackup.py:342
msgid "Destination:"
msgstr "Destinacion :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:307
#: usr/lib/linuxmint/mintbackup/mintbackup.py:343
msgid "Advanced options"
msgstr "Opcions avançadas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:308
#: usr/lib/linuxmint/mintbackup/mintbackup.py:350
msgid "Description:"
msgstr "Descripcion :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:309
msgid "Output:"
msgstr "Sortida :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:310
#: usr/lib/linuxmint/mintbackup/mintbackup.py:344
msgid "Overwrite:"
msgstr "Espotir :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:311
msgid "Confirm integrity"
msgstr "Verificar l'integritat dels salvaments"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:312
msgid "Follow symlinks"
msgstr "Seguir los ligams simbolics"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:313
msgid "Preserve permissions"
msgstr "Preservar las permissions"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:314
msgid "Preserve timestamps"
msgstr "Preservar las datas associadas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:318
msgid "Please select any files or directories you want to exclude"
msgstr "Seleccionatz los fichièrs e repertòris d'exclure"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:319
msgid "Exclude files"
msgstr "Exclure de fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:320
msgid "Exclude directories"
msgstr "Exclure de repertòris"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:321
msgid "Remove"
msgstr "Levar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:325
msgid "Please review the information below before starting the backup"
msgstr "Verificatz las informacions çaijós abans de contunhar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:329
msgid "Please wait while your files are being backed up"
msgstr "Mercé de pacientar pendent que vòstres fichièrs son salvats"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:330
#: usr/lib/linuxmint/mintbackup/mintbackup.py:375
msgid "Backing up:"
msgstr "Salvament en cors :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:334
#: usr/lib/linuxmint/mintbackup/mintbackup.py:379
msgid "The backup is now finished"
msgstr "Lo salvament es d'ara enlà acabat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:338
msgid ""
"Please choose the type of backup to restore, its location and a destination"
msgstr ""
"Seleccionatz un tipe de salvament, sa localizacion e una destinacion per son "
"restabliment"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:339
msgid "Archive"
msgstr "Archiu"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:340
msgid "Directory"
msgstr "Repertòri"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:348
msgid "Please review the information below"
msgstr "Verificatz las informacions çaijós abans de contunhar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:354
msgid "Please wait while your files are being restored"
msgstr "Mercé de pacientar pendent lo restabliment de vòstres fichièrs"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:355
msgid "Restoring:"
msgstr "Restabliment en cors :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:359
msgid "The restoration of the files is now finished"
msgstr "Lo restabliment de vòstres fichièrs es d'ara enlà acabat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:363
msgid "Please choose a destination"
msgstr "Seleccionatz una destinacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:546
msgid "Destination"
msgstr "Destinacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:368
msgid "The list below shows the packages you added to Linux Mint"
msgstr "Los logicials qu'avètz installats apareisson çaijós"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:369
#: usr/lib/linuxmint/mintbackup/mintbackup.py:389
msgid "Select all"
msgstr "Seleccionar tot"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:370
#: usr/lib/linuxmint/mintbackup/mintbackup.py:390
msgid "Deselect all"
msgstr "Deseleccionar tot"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:374
msgid "Please wait while your software selection is being backed up"
msgstr "Mercé de pacientar pendent lo salvament de vòstra lista de logicials"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:383
msgid "Please select a saved software selection"
msgstr "Seleccionatz una lista de logicials"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:384
msgid "Software selection:"
msgstr "Lista de logicials :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:388
msgid "Select the packages you want to install"
msgstr "Seleccionatz los paquets d'installar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:391
msgid "Refresh"
msgstr "Metre a jorn"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:395
msgid "The restoration is now finished"
msgstr "Lo restabliment es d'ara enlà acabat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:396
msgid "Your package selection was restored successfully"
msgstr "Vòstra causida de logicials es estada restablida amb succès"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
msgid "Invalid path"
msgstr "Camin invalid"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:452
#: usr/lib/linuxmint/mintbackup/mintbackup.py:470
#, python-format
msgid "%s is not located within your source directory. Not added."
msgstr "%s es pas inclús dins lo repertòri font. Pas apondut."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:527
msgid "Please choose directories for the source and the destination"
msgstr "Causissètz de repertòris per la font e per la destinacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:530
msgid ""
"Please choose different directories for the source and the destination"
msgstr "Seleccionatz de repertòris font e destinacion diferents"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:545
msgid "Source"
msgstr "Font"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:548
msgid "Description"
msgstr "Descripcion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:552
msgid "Compression"
msgstr "Compression"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:556
msgid "Overwrite destination files"
msgstr "Espotir los fichièrs de destinacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:559
msgid "Exclude"
msgstr "Exclure"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:582
msgid "Please choose a file to restore from"
msgstr "Seleccionatz un fichièr de restablir"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:585
#: usr/lib/linuxmint/mintbackup/mintbackup.py:605
msgid "Please choose a destination directory"
msgstr "Seleccionatz un repertòri de destinacion"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:628
msgid "No packages need to be installed at this time"
msgstr "Pas cap de paquet d'installar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:635
msgid "Please select one or more packages to install"
msgstr "Seleccionatz al mens un paquet d'installar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:703
#: usr/lib/linuxmint/mintbackup/mintbackup.py:704
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1206
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1208
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1514
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1515
msgid "Calculating..."
msgstr "Calcul en cors..."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:719
#, python-format
msgid "Skipping %(skipped_file)s because named pipes are not supported."
msgstr ""
"Fichièr %(skipped_file)s ignorat perque los tubs nomenats son pas suportats."

#: usr/lib/linuxmint/mintbackup/mintbackup.py:761
#: usr/lib/linuxmint/mintbackup/mintbackup.py:801
msgid "Skipping broken link"
msgstr "Ligam copat ignorat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:762
msgid "Broken link"
msgstr "Ligam copat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:765
#: usr/lib/linuxmint/mintbackup/mintbackup.py:805
msgid "Skipping link"
msgstr "Ligam simbolic ignorat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:829
#: usr/lib/linuxmint/mintbackup/mintbackup.py:838
#: usr/lib/linuxmint/mintbackup/mintbackup.py:847
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1267
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1278
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1291
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1355
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1364
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1374
msgid "Skipping identical file"
msgstr "Fichièrs identics ignorats"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:869
#, python-format
msgid ""
"Warning: Some files were not saved, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Atencion : d'unes fichièrs son pas estats salvats (total salvat : "
"%(current_file)d demest  %(total)d)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:873
msgid "An error occurred during the backup"
msgstr "Una error s'es produita pendent lo salvament"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:883
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1560
msgid "The backup was aborted"
msgstr "Lo salvament es estat interromput"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:891
msgid "The backup completed successfully"
msgstr "Lo salvament s'es acabat amb succès"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:982
msgid "Checksum Mismatch:"
msgstr "Diferéncia dins las signaturas sha1sum :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:983
msgid "Checksum Mismatch"
msgstr "Diferéncia dins las signaturas"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1035
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1037
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1066
msgid "Calculating checksum"
msgstr "Calcul de signatura sha1sum"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1112
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1136
msgid "(Not created with the Backup Tool)"
msgstr "(Salvament creat amb una aisina diferenta d'aquela)"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1398
#, python-format
msgid ""
"Warning: Some files were not restored, copied: %(current_file)d files out of "
"%(total)d total"
msgstr ""
"Atencion, certans fichièrs son pas estats restablits : %(current_file)d "
"fichièrs copiats sus un total de %(total)d"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1401
msgid "An error occurred during the restoration"
msgstr "Una error s'es produita pendent lo restabliment"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1412
msgid "The restoration was aborted"
msgstr "Lo restabliment es estat interromput"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1420
msgid "The restoration completed successfully"
msgstr "Lo restabliment s'es acabat amb succès"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1551
msgid "An error occurred during the backup:"
msgstr "I a agut una error pendent la còpia de seguretat :"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1568
msgid "Your software selection was backed up successfully"
msgstr "Vòstra causida de logicials es estada salvada amb succès"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1597
msgid "The specified file is not a valid software selection"
msgstr "Lo fichièr especificat es pas una lista de logicials valida"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1605
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1661
msgid "An error occurred while accessing the file"
msgstr "Una error s'es produita al moment de l'accès al fichièr"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1648
#: usr/lib/linuxmint/mintbackup/mintbackup.py:1654
msgid "Could not locate the package"
msgstr "Lo paquet pòt pas èsser localizat"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1686
msgid "Please wait, this can take some time"
msgstr "Esperatz se vos plai, aquò pòt durar"

#: usr/lib/linuxmint/mintbackup/mintbackup.py:1688
msgid "The installation is complete"
msgstr "L'installacion es acabada"

#: generate_desktop_files.py:26 generate_desktop_files.py:41
msgid "Make a backup of your home directory"
msgstr "Far una còpia de seguretat de vòstre dorsièr personal"

#~ msgid "Cleaning up..."
#~ msgstr "Netejatge..."

#~ msgid "Read error"
#~ msgstr "Error de lectura"

#~ msgid "Hidden paths"
#~ msgstr "Chamins esconduts"

#~ msgid "is not located within your home directory. Not added."
#~ msgstr "es pas dins vòstre dorsier personau. Pas ajostat."

#~ msgid "Included hidden directories"
#~ msgstr "Dorsiers esconduts inclús"

#~ msgid "Included"
#~ msgstr "Inclús"

#~ msgid "Excluded Files and Directories"
#~ msgstr "Fichiers e dorsiers exclús"

#~ msgid "Exclude folders"
#~ msgstr "Dorsiers exclús"

#~ msgid "<big><b>Load data into your home directory</b></big>"
#~ msgstr ""
#~ "<big><b>Charjar de las donadas dins vòstre dorsier personau</b></big>"

#~ msgid "View content"
#~ msgstr "Veire lo contengut"

#~ msgid "Backup successful"
#~ msgstr "Còpia de seguretat realizada"

#~ msgid "Your home directory was successfully backed-up into"
#~ msgstr "Vòstre dorsier personau se copiet plan dins"

#~ msgid "Restoration successful"
#~ msgstr "Restauracion reüssida"

#~ msgid "Your backup was successfully restored"
#~ msgstr "Vòstra còpia de seguretat fuguet restaurada emb succes"

#~ msgid "Restore"
#~ msgstr "Restablir"

#~ msgid "Backup"
#~ msgstr "Salvar"

#~ msgid "Archiving your home directory..."
#~ msgstr "Archivar vòstre dorsièr personal..."

#~ msgid "Backup failed"
#~ msgstr "Fracàs del salvament"

#~ msgid "Restoration failed"
#~ msgstr "Fracàs del restabliment"

#~ msgid "Copying the backup file into your home directory..."
#~ msgstr "Còpia del fichièr de salvament dins vòstre dorsièr personal…"

#~ msgid "Overwrite existing files"
#~ msgstr "Espotir los fichièrs existents"

#~ msgid "Opening the backup archive..."
#~ msgstr "Dobertura de l'archiu de salvament…"

#~ msgid "Restore your personal data from this backup"
#~ msgstr "Restablir vòstras donadas personalas dempuèi aqueste salvament"

#~ msgid "Extracting the content of the backup into your home directory..."
#~ msgstr "Extraccion del contengut del salvament dins vòstre dorsièr personal…"

#~ msgid "An error occurred while restoring the backup archive:"
#~ msgstr "Una error s'es producha pendent lo restabliment del salvament :"

#~ msgid "An error occurred while opening the backup:"
#~ msgstr "Una error s'es producha pendent la dobertura del salvament :"
